src/site/layouts/office.drupal.liquid

```diff
@@ -0,0 +1,102 @@
+{% assign entryname = "public-outreach-materials" %}
```

Comment:
This value corresponds to [this](https://github.com/department-of-veterans-affairs/vets-website/pull/10020/files#diff-c84cbc160ef290ec769851c5e766617fR4) value in the `manifest.json`. The `manifest.json` has another property, [`entryFile`](https://github.com/department-of-veterans-affairs/vets-website/pull/10020/files#diff-c84cbc160ef290ec769851c5e766617fR3), that sets up all of this code as its own JS app.